### DISCUSSION TITLE
FIPS 140-2 compliance?

### DISCUSSION THREAD

**Discussion by PathfinderNetworks** [Ylianst/MeshCentral#4124] [General]
One of our customers, a local police agency, is currently doing their CJIS audit. The auditor is expecting all systems to be FIPS 140-2 compliant (which is a debatable issue all on its own).  
Does MeshCentral use cryptographic modules that are FIPS 140-2 compliant?

**Comment by PathfinderNetworks** [comment]
It looks like, at the very least, OpenSSL is FIPS 140-2 compliant.  I can't recall, does MeshCentral happen to utilize OpenSSL?  If so, that would satisfy this auditors requirements.

**Comment by krayon007** [comment]
Yes. Both the Server and Agent for Mesh Central uses OpenSSL. The agent is currently using 1.1.1n, and is regularly updated to the latest.

For the server, the version that it uses depends on which Node version you are updated to, but Ylian can correct me if I'm wrong.

**Comment by PathfinderNetworks** [comment]
Excellent. Thank you.
I was fairly certain this was the case as I recall discussions about the agent and OpenSSL within the last year or so.  
I am at version 18.x for Node so I should be good to go there as well. 

I appreciate it!
